google/cloud/aiplatform/training_jobs.py

```diff
@@ -5091,13 +5672,22 @@ def _run(
         model_tbt.display_name = model_display_name or self._display_name
         model_tbt.labels = model_labels or self._labels
 
+        # AutoMLVideo does not support validation, so pass in '-' if any other filter split is provided.
+        validation_filter_split = (
+            "-"
+            if any([training_filter_split is not None, test_filter_split is not None])
```

Comment:
Since all filter splits are required I think this should be all.

Reply:
Sure, the result would be the same though as it will error out later unless all are set.